1-mandatory/1-writers.js

```diff
@@ -71,8 +88,20 @@ Exercise 3:
   "Hi, my name is {firstName} {lastName}. I am {age} years old."
 */
 
+function getAliveWritersInTheirForties(writer) {
+  return writer.age >= 40 && writer.age <= 49 && writer.alive === true;
+}
+
 function logAliveWritersInTheirForties() {
-  // write your code here
+  let aliveWritersInTheirForties = writers.filter(
+    getAliveWritersInTheirForties
+  );
+
+  aliveWritersInTheirForties.map((writer) => {
+    console.log(
+      `Hi, my name is ${writer.firstName} ${writer.lastName}. I am ${writer.age} years old.`
+    );
+  });
 }
```

Comment:
maybe having all this code under one function would be easier to read? what do you think?